### PR TITLE
Fix typo in StandardCursorType.

### LINKED ISSUE #2784: Misnamed Cursor - BottomSize
I believe `StandardCursorType.BottomSize` is misnamed, I think it is intended to be `BottomSide` to match the other directions.

https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Input/Cursors.cs#L31

### PR DESCRIPTION
**Pull request by grokys**
## What does the pull request do?

Fixes a typo in `StandardCursorType` as described in #2784

## Fixed issues

Fixes #2784

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]

File: src/Avalonia.Input/Cursors.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public enum StandardCursorType
         AppStarting,
         Help,
         TopSide,
-        BottomSize,
+        BottomSide,
```
Comment (kekekeks) [inline src/Avalonia.Input/Cursors.cs]: Add
```cs
[Obsolete("Use BottomSide")]
ButtomSize = BottomSide,

```

Comment (ahopper) [reply src/Avalonia.Input/Cursors.cs]: typo should be BottomSize

Comment (lindexi) [reply src/Avalonia.Input/Cursors.cs]: @kekekeks This enum without number that we should not add value optionally.

Comment (kekekeks) [reply src/Avalonia.Input/Cursors.cs]: @ahopper @lindexi we shouldn't just break people's code by randomly renaming stuff. The common practice is to make the old naming obsolete and remove it after several versions.

Comment (ahopper) [reply src/Avalonia.Input/Cursors.cs]: very true but you have spelled BottonSize as ButtomSize

Comment (kekekeks) [reply src/Avalonia.Input/Cursors.cs]: Ah, ok

**Comment by grokys** [comment]
Added an obsolete alias for old typo'd value.

**Pull request merged**